### PR TITLE
feat: apply django migrations inside the bahis-serve dockerfile

### PR DESCRIPTION
**Pull request by ChasNelson1990**
## Description

Now that migrations are working and committed to the repo, we need to make sure we apply them when we deploy bahis-infra.

So the protocol should be:

1. developer makes changes to models in bahis-serve
2. developer runs `makemigrations` in their dev environment, committing migrations to their feature branch
3. reviewer ensure migrations are in any PR and are documented in the PR description
4. bahis-infra automatically applies migrations on the next deploy

relates road86/bahis-serve#25

## Checklist

- [x] I have read the road86 Contribution Guide.
- [x] I have checked all commit message styles match the requested structure.
- [x] My code follows the style guidelines of this project.
- [x] I have performed a self-review of my own code.
- [x] I have assigned at least one reviewer.
- [ ] New and existing unit tests pass locally with my changes.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [x] My changes generate no new warnings.
- [ ] I have made corresponding changes to the documentation.
- [x] New database changes have been committed.

### CODE REVIEW

**Comment by shaarful** [comment]
@ChasNelson1990 It is creating lots of new tables in the public scheme. They are duplicates like logger_instance, api_project, api_team and lots...

**Comment by shaarful** [comment]
I am still reviewing it

**Comment by ChasNelson1990** [comment]
@sharful09 - maybe you're right then and we just do `migrate bh_module`?

**Comment by shaarful** [comment]
@ChasNelson1990 I am getting problem with log in through bahis desk. after migrate all

**Comment by ChasNelson1990** [comment]
> @ChasNelson1990 I am getting problem with log in through bahis desk. after migrate all

Ah, yes, I didn't try bahis-desk. Wow, I didn't do very well with this one lol.

How about with just `migrate bh_module`?

**Comment by shaarful** [comment]
> > @ChasNelson1990 I am getting problem with log in through bahis desk. after migrate all
> 
> Ah, yes, I didn't try bahis-desk. Wow, I didn't do very well with this one lol.
> 
> How about with just `migrate bh_module`?

that is working fine for me.

File: deployment/bahisweb/Dockerfile [hunk 1]
```diff
@@ -76,4 +76,4 @@ ENV DJANGO_DEBUG="False"
 
 #CMD ["./uwsgi", "bahis.ini"]
 #we need 10s delay to give databases time to initialise
-CMD sleep 10 && python /${SRC_DIR}/kobocat/manage.py collectstatic --noinput && ./celeryd_bahis start && ./uwsgi bahis.ini
+CMD sleep 10 && python /${SRC_DIR}/kobocat/manage.py collectstatic --noinput && python /${SRC_DIR}/kobocat/manage.py migrate && ./celeryd_bahis start && ./uwsgi bahis.ini
```
Comment (shaarful) [inline deployment/bahisweb/Dockerfile]: Django migration creates lots of duplicate tables. We can stick to no migration at the startup container. if someone makes new model then he will be responsible for doing the migration for that particular app.

Comment (ChasNelson1990) [reply deployment/bahisweb/Dockerfile:79]: Okay then, let's close this PR without merging.

Comment (ChasNelson1990) [reply deployment/bahisweb/Dockerfile]: Changed to `migrate bh_module` is that what you meant?

Comment (shaarful) [reply deployment/bahisweb/Dockerfile]: I meant we do not need to run any kind of migration at the startup of the container. It is only one-time job. If someone makes any changes in the model then we can manually migrate it from the terminal of that container.

Comment (ChasNelson1990) [reply deployment/bahisweb/Dockerfile]: Ah, got ya. Let's close this PR without merging then.

**Pull request closed without merging**